.ruby-style.yml

```diff
@@ -24,41 +24,321 @@ AllCops:
   StyleGuideCopsOnly: false
   DisabledByDefault: true
 
-Style/AccessModifierIndentation:
+Layout/AccessModifierIndentation:
```

Comment:
For reference, this is documented in the [rubocop changelog](https://github.com/bbatsov/rubocop/blob/master/CHANGELOG.md#changes-2).